homeassistant/components/switch/zigate.py

```diff
@@ -0,0 +1,124 @@
+"""
+ZiGate platform.
+
+For more details about this platform, please refer to the documentation
+https://home-assistant.io/components/ZiGate/
+"""
+import logging
+from homeassistant.components.switch import SwitchDevice
+
+DOMAIN = 'zigate'
```

Comment:
This platform belongs to the switch domain. Remove this constant and instead import the zigate domain `as ZIGATE_DOMAIN`.

Reply:
Ok, I understand but since during dev it's a custom_components, something like :
```
from homeassistant.components.zigate import DOMAIN as ZIGATE_DOMAIN
```
will failed...
any tips ?
Could I do :
```
try:
    from homeassistant.components.zigate import DOMAIN as ZIGATE_DOMAIN
except:
    from homeassistant.custom_components.zigate import DOMAIN as ZIGATE_DOMAIN
```

Reply:
I don't know. I don't think it's good to develop while testing it as a custom component. You should test it as it will be implemented.

Reply:
ok, thanks